File: 2293-Minimum Number of Steps to Make Two Strings Anagram II.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        c = defaultdict(int)
        
        for i in s:
            c[i] += 1
        
        for i in t:
            c[i] -= 1
        
        res = 0
        for i, v in c.items():
            res += abs(v)
        
        return res

File: 2293-Minimum Number of Steps to Make Two Strings Anagram II.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        int c[26] = {0};
        
        for (char i : s)
            c[i-'a'] += 1;
        
        for (char i : t)
            c[i-'a'] -= 1;
        
        int res = 0;
        for(int i = 0; i < 26; i++){
            res += abs(c[i]);
        }
        return res;
    }
};